pubtools/pulplib/_impl/model/repository/base.py

```diff
@@ -303,6 +303,36 @@ class Repository(PulpObject, Deletable):
     .. versionadded:: 2.29.0
     """
 
+    include_in_download_service = pulp_attrib(
+        default=False,
+        type=bool,
+        pulp_field="notes.include_in_download_service",
+        pulp_py_converter=lambda x: x == "True",
+        py_pulp_converter=str,
+    )
+    """Flag indicating whether the repository is visible in production instance
+    of download service.
+
+    Although the value looks like a boolean, it is always stored as string in Pulp.
```

Comment:
I'd rather not even mention this as the library is supposed to encapsulate entirely how the fields are serialized in Pulp.  If you stick with this library you'll never even see this quirk.  In the doc strings for other fields we don't generally document how they are being serialized (e.g. signing_keys, product_versions).

Reply:
The comment should have served as an explanation for the next person reading the codebase and thinking "_Why is a boolean value interpreted as a string? This is surely a mistake. Let me fix it._". On the other hand you're right that the lists you mention are also stored as json strings without anyone raising an eyebrow (although in the boolean case the eyebrow should be risen much higher, especially because we have also other boolean values with are proper booleans)